src/server/services/catalog.service.test.ts

```
@@ -0,0 +1,88 @@
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+import { Role, type PrismaClient } from '@orm/client'
```

ESLint

[error] `@orm/client` import should occur before import of `vitest`

(import/order)